performance/src/efficiency/nseed_performance_writer.cpp

```diff
@@ -0,0 +1,147 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022-2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#include <memory>
+#include <optional>
+#include <traccc/efficiency/nseed_performance_writer.hpp>
+#include <traccc/efficiency/track_filter.hpp>
+#include <traccc/efficiency/track_matcher.hpp>
+
+namespace traccc {
+nseed_performance_writer::nseed_performance_writer(
+    const std::string& prefix, std::unique_ptr<track_filter>&& filter,
+    std::unique_ptr<track_matcher>&& matcher)
+    : _prefix(prefix),
+      _filter(std::forward<std::unique_ptr<track_filter>>(filter)),
+      _matcher(std::forward<std::unique_ptr<track_matcher>>(matcher)) {}
+
+void nseed_performance_writer::initialize() {
```

Comment:
Actually I entirely forgot about this comment I had. Would it not be safer to call these in the objects constructor and destructor?

Reply:
meaning initialize() and finalize()